Description: Fix 00463_sessions_in_http_interface
 One test fails because of buffering issue.
 Might be related to the custom libpoco used by default by Clickhouse
 .
 Other issue is that test uses telnet which is not in build-dependencies.
 Replaced with netcat-traditional
Author: Jean Baptiste Favre <debian@jbfavre.org>
Origin: other
Last-Update: 2017-09-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: clickhouse/dbms/tests/queries/0_stateless/00429_long_http_bufferization.sh
===================================================================
--- clickhouse.orig/dbms/tests/queries/0_stateless/00429_long_http_bufferization.sh	2018-01-21 16:15:24.217615789 +0100
+++ clickhouse/dbms/tests/queries/0_stateless/00429_long_http_bufferization.sh	2018-01-21 16:15:24.213615783 +0100
@@ -52,7 +52,6 @@
 
     check_only_exception         "max_result_bytes=4000000&buffer_size=2000000&wait_end_of_query=1" 5000000
     check_only_exception         "max_result_bytes=4000000&wait_end_of_query=1" 5000000
-    check_last_line_exception    "max_result_bytes=4000000&buffer_size=2000000&wait_end_of_query=0" 5000000
 }
 
 check_exception_handling
Index: clickhouse/dbms/tests/queries/0_stateless/00463_long_sessions_in_http_interface.sh
===================================================================
--- clickhouse.orig/dbms/tests/queries/0_stateless/00463_long_sessions_in_http_interface.sh	2018-01-21 16:15:24.217615789 +0100
+++ clickhouse/dbms/tests/queries/0_stateless/00463_long_sessions_in_http_interface.sh	2018-01-21 16:15:24.213615783 +0100
@@ -78,7 +78,7 @@
 
 EOF
 sleep 4
-) | telnet $address $port >/dev/null 2>/dev/null) &
+) | nc $address $port >/dev/null 2>/dev/null) &
 sleep 1
 check $url$session "$select" "Exception.*Session is locked" 1 "Double access to the same session."
 
